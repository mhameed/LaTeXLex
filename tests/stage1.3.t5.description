author: ehc
date: 09/10/07
description: test that \mathcal{A}, {\mathcal A}, \cal{A}, {\cal A} all produce \calA and similar for scr and frak
