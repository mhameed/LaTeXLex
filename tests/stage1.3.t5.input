Here is some text and now some maths
${\mathcal A}, \mathcal{A}, \cal{A}, {\cal A}$
${\mathscr A}, \mathscr{A}, \scr{A}, {\scr A}$
${\mathfrak A}, \mathfrak{A}, \frak{A}, {\frak A}$
${\mathbb A}, \mathbb{A}, \bb{A}, {\bb A}$

And here is a harder one in a sentence \mathcal A
\mathcal A is interesting and then
$$
\mathcal A
$$
and $$\mathcal A$$ and $\mathcal A$

Some harder examples:
$\mathcal{A B C}$
$\mathcal{A^{\#}}$
$\mathcal{{A}} \mathcal{.A.}$
There is some question regarding what we want the output to be in these cases.

$\mathbb 1$
$\mathcal 1$
$\mathbb{10}$
$\mathbb 10$
