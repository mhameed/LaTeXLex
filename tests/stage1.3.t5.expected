Here is some text and now some maths
$\calA, \calA, \calA, \calA$
$\scrA, \scrA, \scrA, \scrA$
$\frakA, \frakA, \frakA, \frakA$
$\bbA, \bbA, \bbA, \bbA$

And here is a harder one in a sentence \calA
\calA is interesting and then
$$
\calA
$$
and $$\calA$$ and $\calA$

Some harder examples:
$\calA \calB \calC$
$\calA^{\#}$
${\calA} .\calA.$
There is some question regarding what we want the output to be in these cases.

$\bb1$
$1$
$\bb1\bb0$
$\bb10$
